src/test/txpackage_tests.cpp

```diff
@@ -368,6 +545,101 @@ BOOST_FIXTURE_TEST_CASE(package_submission_tests, TestChain100Setup)
         BOOST_CHECK(m_node.mempool->exists(GenTxid::Txid(tx_parent->GetHash())));
         BOOST_CHECK(m_node.mempool->exists(GenTxid::Txid(tx_child->GetHash())));
     }
+
+    // do not allow parents to pay for children
+    {
+        Package package_ppfc;
+        // Diamond shape:
+        //
+        //     grandparent
+        //       5679sat
```

Comment:
diamond keeps going "out of sync", maybe just have variable names that people can find?